service/frontend/workflowHandler.go

```diff
@@ -133,6 +133,7 @@ var (
 	errClusterNameNotSet                          = &types.BadRequestError{Message: "Cluster name is not set."}
 	errEmptyReplicationInfo                       = &types.BadRequestError{Message: "Replication task info is not set."}
 	errEmptyQueueType                             = &types.BadRequestError{Message: "Queue type is not set."}
+	errDomainInLockdown                           = &types.BadRequestError{Message: "Domain is not accepting fail overs at this time."}
```

Comment:
"Domain is not accepting fail overs due to lockdown." I think it's better to tell users there is a lockdown.